Assets/Project/Modules/PlayerAnchor/Scripts/AbilityUnlock/Unlocker/ChainedOrb/AbilityUnlockerChainedOrbView.cs

```diff
@@ -53,25 +62,36 @@ private void OnDestroy()
 
             _orbHitPS = Instantiate(typeViewData.OrbHitParticlesPrefab, _orbHitParticlesHolder);
             _pickAbilityPS = Instantiate(typeViewData.PickAbilityParticlesPrefab, _pickAbilityParticlesHolder);
+
+            _particleFactory = ServiceLocator.Instance.GetService<IParticleFactory>();
         }
 
 
         public async UniTaskVoid PlayIdleAnimation()
         {
             _movingChainsSoundId = _audio.StartPlayingMovingChainsSound(gameObject);
+
+            _orbitalChains0.DOBlendableLocalRotateBy(Vector3.up, 180).SetLoops(-1);
+            _orbitalChains0.DOBlendableLocalRotateBy(Vector3.up, 60).SetLoops(-1);
+            _orbitalChains0.DOBlendableLocalRotateBy(Vector3.up, 120).SetLoops(-1);
```

Comment:
These numbers (180, 60, 120) sould be parameters inside the `ChainedOrbViewConfig` class